### PR TITLE
Add description about admin updating the Credit Limit

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds description about admin updating the Credit Limit

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/customers/credit-company-activity.html

### CODE REVIEW

File: src/customers/credit-company-activity.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ The [Company Credit]({% link customers/account-dashboard-company-credit.md %}) s
 ![]({% link images/images-b2b/company-credit-reimbursements-grid.png %}){: .zoom}
 _Company Credit activity_
 
+## Admin Updating the Credit Limit
+
+When the customer makes the payment for his outstanding credit limit to the admin, then the admin needs to update the customer’s detail in the panel:
```
Comment (leslietilling) [inline src/customers/credit-company-activity.md:15]:
```suggestion
When the customer makes the payment for their outstanding credit to the merchant, a store administrator must then update the customer credit information in the Admin.
```

File: src/customers/credit-company-activity.md [hunk 2]
```diff
@@ -10,6 +10,22 @@ The [Company Credit]({% link customers/account-dashboard-company-credit.md %}) s
...
+
+1. On the _Admin_ sidebar, go to **Customers > Companies**.
+
+1. Find the company in the grid and open in _Edit_ mode.
+
+1. Expand the **Company Credit** tab.
```
Comment (leslietilling) [inline src/customers/credit-company-activity.md:21]:
```suggestion
1. Expand the **Company Credit** section.
```

File: src/customers/credit-company-activity.md [hunk 3]
```diff
@@ -10,6 +10,22 @@ The [Company Credit]({% link customers/account-dashboard-company-credit.md %}) s
...
+
+1. Set the new value to the **Credit Limit** field.
```
Comment (leslietilling) [inline src/customers/credit-company-activity.md:23]:
```suggestion
1. For **Credit Limit**, enter the new value.
```

File: src/customers/credit-company-activity.md [hunk 4]
```diff
@@ -10,6 +10,22 @@ The [Company Credit]({% link customers/account-dashboard-company-credit.md %}) s
...
+
+1. Change the other values as needed.
+
+1. When ready, click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/customers/credit-company-activity.md:27]:
```suggestion
1. When updates are complete, click <span class="btn">Save</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing. I have implemented your suggestions, please approve my PR 😊

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**